File: Version0.2/Julia/Postprocessing/generatecode.jl
function  generatecode(app,mesh)

    # search compilers and set options
    app = Gencode.setcompilers(app);

    # generate input files and store them in datain folder
    app, mesh, master, dmd = preprocessing(app,mesh);
    
    # generate source codes and store them in app folder
    Gencode.gencode(app);
        
    return app,mesh,master,dmd
    
end
    

File: frontends/Python/Postprocessing/generatecode.py
import Preprocessing, Gencode

def generatecode(pde,mesh):

    # search compilers and set options
    pde = Gencode.setcompilers(pde);

    # generate input files and store them in datain folder
    pde, mesh, master, dmd = Preprocessing.preprocessing(pde,mesh);

    # generate source codes and store them in app folder
    Gencode.gencode(pde);

    return pde,mesh,master,dmd
